src/Symfony/Component/Form/Resources/translations/validators.af.xlf

```diff
@@ -0,0 +1,139 @@
+            </trans-unit>
+            <trans-unit id="99">
+                <source>This value is not a valid HTML5 color.</source>
+                <target>Hierdie waarde is nie 'n geldige HTML5 kleur nie.</target>
+            </trans-unit>
+            <trans-unit id="100">
+                <source>Please enter a valid birthdate.</source>
+                <target>Voer asseblief 'n geldige geboortedatum in.</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>The selected choice is invalid.</source>
+                <target>Die gekiesde opsie is nie geldig nie.</target>
+            </trans-unit>
+            <trans-unit id="102">
+                <source>The collection is invalid.</source>
+                <target>Die versameling is nie geldig nie.</target>
+            </trans-unit>
+            <trans-unit id="103">
+                <source>Please select a valid color.</source>
+                <target>Kies asseblief 'n geldige kleur.</target>
+            </trans-unit>
+            <trans-unit id="104">
+                <source>Please select a valid country.</source>
+                <target>Kies asseblief 'n geldige land.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>Please select a valid currency.</source>
+                <target>Kies asseblief 'n geldige geldeenheid.</target>
+            </trans-unit>
+            <trans-unit id="106">
+                <source>Please choose a valid date interval.</source>
+                <target>Kies asseblief 'n geldige datum interval.</target>
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Please enter a valid date and time.</source>
+                <target>Voer asseblilef 'n geldige datum en tyd in.</target>
+            </trans-unit>
+            <trans-unit id="108">
+                <source>Please enter a valid date.</source>
+                <target>Voer asseblief 'n geldige datum in.</target>
+            </trans-unit>
+            <trans-unit id="109">
+                <source>Please select a valid file.</source>
+                <target>Kies asseblief 'n geldige lêer.</target>
+            </trans-unit>
+            <trans-unit id="110">
+                <source>The hidden field is invalid.</source>
+                <target>Die versteekte veld is nie geldig nie.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>Voer asseblief 'n geldige heeltal in.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Kies assblief 'n geldige taal.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Voer assebliefn 'n geldige landstreek in.</target>
```

Comment:
Does this refer to a locale like "en", "af", "fr" etc?

Reply:
There isn't really anything with the same meaning in Afrikaans, this refers more to something like a `Language area`, which might not be the same thing. I see some other translations just left it as `locale`, so I made the same change here.

Reply:
Okey. 
As long as it makes sense. 

Thank you 👍